### PR TITLE
Create better quality thumbnails

### LINKED ISSUE #178: Create better quality thumbnails
I recently tried making a tag from an existing image, and the produced image was extremely low quality even though the source image was high quality.

## Reproduction steps
1. Upload a high quality image to an officer's page
2. Click on the image
3. Click "Tag this image"
4. Add a tag that covers most of the image, select a different OO officer ID than the original
5. Observe that the new generated image is tiny (my personal example went from 8MB to 17kb)

### PR DESCRIPTION
**Pull request by sea-kelp**
## Description of Changes
Fixes #178.

Bump the thumbnail bounding box to 1000px x 1000px and always save the thumbnail as jpeg.

Let me know what you think about the new bounding box dimensions

## Notes for Deployment
None!

## Screenshots (if appropriate)
Original picture (1.2mb):

Old thumbnail: (15.1kb)

New thumbnail: (102.3kb, about 6.8x larger than older version)

## Tests and linting

 - [x] I have rebased my changes on `main`

 - [x] `just lint` passes

 - [x] `just test` passes

### CODE REVIEW

File: OpenOversight/app/utils.py [hunk 1]
```diff
@@ -565,36 +565,34 @@ def create_description(self, form):
...
-        if image_type in ("jp2", "j2k", "jpf", "jpx", "jpm", "mj2"):
-            image_type = "jpeg2000"
-        elif image_type in ("jpg", "jpeg", "jpe", "jif", "jfif", "jfi"):
-            image_type = "jpeg"
-        elif image_type in ("tif", "tiff"):
-            image_type = "tiff"
     pimage = Pimage.open(image_buf)
 
-    SIZE = 300, 300
+    SIZE = 1000, 1000
```
Comment (AetherUnbound) [inline OpenOversight/app/utils.py:579]: Could we rename this to `THUMBNAIL_SIZE` and move this to the top of the file so it functions more as a module level variable?

File: OpenOversight/app/utils.py [hunk 2]
```diff
@@ -565,36 +565,34 @@ def create_description(self, form):
...
     if not crop_data and pimage.size[0] < SIZE[0] and pimage.size[1] < SIZE[1]:
         return image
 
+    # Crops image to face and resizes to bounding box if still too big
```
Comment (AetherUnbound) [inline OpenOversight/app/utils.py:583]: Thank you for these comments!

**Review by AetherUnbound — APPROVED** [review]
This is **such** a needed fix, thank you! Tested locally and works as expected!

File: OpenOversight/app/utils.py (hunk 1 above)
Comment (sea-kelp) [reply OpenOversight/app/utils.py]: Done!

**Pull request merged**